modules/appeals_api/config/routes.rb

```diff
@@ -22,12 +22,16 @@
       end
       namespace :notice_of_disagreements do
         get 'contestable_issues', to: 'contestable_issues#index'
-        resources :evidence_submissions, only: %i[show]
+        resources :evidence_submissions, only: %i[show] do
+          collection do
```

Comment:
Is this a collection? Are we set up to handle multiple documents at a time? Perhaps we should require the client to submit a single request per document?

Reply:
^ this was me misunderstanding collection it seems. I'll refactor